### PR TITLE
added task input

### PR DESCRIPTION
**Pull request by DinaAyman03**
I changed in the javascript as i used MVC to make the delete and check buttons of the Todo list and i also changed a little bit in the CSS

### CODE REVIEW

**Review by youssefibrahim22 — COMMENTED** [review]
image

please take care from the naming conventions
such as the (uid) attribute which should be (uId) following camel case

**Review by ManarMahmoud17 — COMMENTED** [review]
It looks like you're trying to handle adding a task to a list, and you need to manage the case when the list is empty. If the list is empty, you want to start with an ID of 1. Here's a corrected version of the line:

id: model.getTodos().length ? model.getTodos()[model.getTodos().length - 1].id + 1 : 1

File: index.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+                           <button onclick="controller.handleDelete(this)" class="circle delete">
+                           <i class="fas fa-trash"></i>
+                           </button></li>`;
+    const todoListElem = document.getElementById("todosList");
+
+    todoListElem.innerHTML += todoElem;
+    return;
+  },
+  delteObject: function(uid) {
```
Comment (Sara-Hesham-Mohammed) [inline index.js:64]: Small typo in the word delete. Perhaps change for clarity and to avoid future confusion.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  },
+  deleteTodo: function (uid) {
+    this.todos=this.todos.filter((todo) => todo.id !== parseInt(uid));
+    console.log(model.todos);
+    view.delteObject(uid);
+  },
+  checkTodo: function(uid) {
+    const todo =this.todos.find((todo) => todo.id === parseInt(uid));
+    todo.completed = !todo.completed;
+    console.log(this.todos);
```
Comment (Sara-Hesham-Mohammed) [inline index.js:40]: print statements should be removed

File: index.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  addTodo: function (todo) {
+    this.todos.push(todo);
+    view.renderTodo(todo);
+  },
+  getTodos: function () {
+    return this.todos;
+  },
+  deleteTodo: function (uid) {
+    this.todos=this.todos.filter((todo) => todo.id !== parseInt(uid));
+    console.log(model.todos);
```
Comment (Sara-Hesham-Mohammed) [inline index.js:34]: print statements should be removed

**Review by rawanramadanseddik — APPROVED** [review]
More than perfecto!

**Review by Sara-Hesham-Mohammed — CHANGES_REQUESTED** [review]
**Great work**, however very small changes are needed for clarity.

**Review by DaliaAmr — APPROVED** [review]
remove only all console.log but over all everything is good

**Pull request merged**